packages/framework/src/Form/Admin/Product/Parameter/ProductParameterValueFormType.php

```diff
@@ -21,9 +22,12 @@ class ProductParameterValueFormType extends AbstractType
 
     /**
      * @param \Shopsys\FrameworkBundle\Model\Product\Parameter\ParameterFacade $parameterFacade
+     * @param \Shopsys\FrameworkBundle\Model\Localization\Localization $localization
      */
-    public function __construct(private readonly ParameterFacade $parameterFacade)
-    {
+    public function __construct(
+        private readonly ParameterFacade $parameterFacade,
+        private readonly Localization $localization,
```

Comment:
Missing an upgrade notes?

Reply:
the FormTypes are not meant to be extended via inheritance